pkg/repo/chartrepo_test.go

```diff
@@ -145,6 +146,14 @@ func verifyIndex(t *testing.T, actual *IndexFile) {
 				},
 			},
 		},
+		"universe/zarthal": {
+			{
+				Metadata: &chart.Metadata{
+					Name:    "universe/zarthal",
```

Comment:
I'm not clear on why the name would be "zarthal" if the file is "universe/zarthal-1.0.0.tgz"? I'm sorry if I'm missing something here, I'm having a hard time keeping the whole context of the conversation in my head, as it's gone on so long (which I recognise is entirely my fault.)